src/instance/misc.js

```diff
@@ -39,6 +39,14 @@ exports._applyFilters = function (value, oldValue, filters, write) {
   return value
 }
 
+exports._bubbleChanges = function (value, expression) {
```

Comment:
When I saw `scope._bubbleChanges` it's a bit confusing, because this is a v-for-scope-only thing, but is attached to Vue's prototype, which imo is the wrong place. It only works because of the prototypal inheritance.